dts/xtensa/intel/intel_adsp_ace15_mtpm.dtsi

```diff
@@ -379,5 +379,59 @@
 			lps = <&lps>;
 			bit-position = <13>;
 		};
+
+		/*
+		 * Can we fix this somehow?
```

Comment:
you could perhaps mimic the cpu definition above in the same file: 
"cpu0: cpu@0" and same for the reg entry...

Reply:
certainly we should not merge this comment ;) 
TODO looks more professionally

Reply:
This comment is only for review-time to get comments about this part :) will be removed ofc